.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+# Pull Request
```

Comment:
Pull requests don't really need to be titled Pull Request.  They don't need a title in the body at all.